### DISCUSSION TITLE
DIFFERENTIAL MARKER PEAKS BETWEEN SAMPLES (Compare cluster peaks between samples)

### DISCUSSION THREAD

**Discussion by matteozoia4** [GreenleafLab/ArchR#1622] [Questions / Documentation]
-> I have a Project containing 2 samples ("Sample" cellColData column)
-> They shares identical  clusters "C1-21" (same cell type).

 **How can I perform a differential analysis comparing C3 (Cluster-3) of Sample 1 with C3 of sample-2?**
    
 Ps: _I would ideally want to be able to confront each same Cluster between samples._

```

markerDiff <- getMarkerFeatures(
  ArchRProj = Project,
  useMatrix = "PeakMatrix",
  groupBy = "Clusters", 
  testMethod = "wilcoxon",
  bias = c("TSSEnrichment", "log10(nFrags)"),
  useGroups = c("C3"),
  bgdGroups = c("C1","C2","C4", "C5", "C6", "C7", "C8", "C9", "C10", "C11", "C12", "C13", "C14", "C15",
                "C16", "C17", "C18", "C19", "C20", "C21")
)
```

Many thanks for the help,

MZ

**Comment by rcorces** [comment]
create a new column in cellColData that represents sample_x_cluster and use that in getMarkerFeatures as discussed here:
https://www.archrproject.com/bookdown/pairwise-testing-between-groups.html
